_rules/scrollable-element-keyboard-accessible-0ssw9k.md

```diff
@@ -160,7 +162,7 @@ This [horizontally scrollable][scrollable] `section` element is not included in
 
 #### Inapplicable Example 1
 
-This `section` element has a [computed][] [overflow][] of `visible`.
+This `section` element is not [scrollable][] because it has a [scroll distance][scrollable] of 0 in both directions.
```

Comment:
Revert

```suggestion
This `section` element has a [computed][] [overflow][] of `visible`.
```

Reply:
A survey comment was to use terms from the Applicability to describe why this example is inapplicable. Is there a way to say "computed overflow of visible" using terms from Applicability?

Reply:
Well... the descriptions of 1 and 2 are the same as you're suggesting them. I'm not exactly sure what you're looking for. The thing example 1 does is that the container isn't tall enough to contain its content, but because it has overflow: visible, the content doesn't become scrollable. It just renders outside the container. Inapplicable example 2 on the other hand, isn't scrollable because the content isn't long enough. Because of overflow:auto on example 2, if it had more content, it would be scrollable.

The current descriptions are saying that... maybe this would help?

```suggestion
This `section` element has a [computed][] [overflow][] of `visible`. Because of this its [scroll distance][scrollable] will be 0, regardless of the element's content.
```

And then for example 2 you could use:

```md
This `section` element has a [scroll distance][scrollable] of 0 in both directions because the content is smaller than the `section` element.
```